nomad/blocked_evals.go

```diff
@@ -0,0 +1,303 @@
+package nomad
+
+import (
+	"sync"
+	"time"
+
+	"github.com/armon/go-metrics"
+	"github.com/hashicorp/nomad/nomad/structs"
+)
+
+const (
+	// unblockBuffer is the buffer size for the unblock channel.
```

Comment:
Add a comment that this must be large enough to prevent a stall in the FSM which applies back pressure to Raft